### ISSUE TITLE
Tuning lm_weight, word_score and beam_size

### ISSUE THREAD

**Issue by mohitsharma29** [mailong25/self-supervised-speech-recognition#18]
Hey

How do you recommend we tune the parameters in transcribe function: lm_weight, word_score, and beam_size? Normally with things like Deepspeech 2, we use its logits to tune this, but how about wav2vec?

Thanks

**Comment by mailong25** [comment]
You can prepare a test set and perform a simple grid search for hyperparameter tuning

**Comment by mohitsharma29** [comment]
Yes, that's the last resort. That will be very slow because every time you infer using the acoustic model. With DS2's grid search, they freeze the logits and just search over the LM params, making the thing a lot faster. I was wondering if we can do something like that here.

**Comment by DatNgoBK** [comment]
I also meet this issue
